src/pages/trader-tools/pnl-for-margin/_pnl-margin-calculator.tsx

```diff
@@ -306,14 +303,13 @@ const PnlMarginCalculator = () => {
                                     />
                                 )
 
-                                const PointValueInput = ({ field }: { field: React.ReactNode }) => (
+                                const PointValueInput = () => (
                                     <Input
-                                        {...field}
                                         id="pointValue"
                                         type="text"
                                         label={localize('Point value')}
                                         autoComplete="off"
-                                        error={touched.pointValue && errors.pointValue}
+                                        error={touched.pointValue && errors.pointValue.toString()}
```

Comment:
please use optional chaining to prevent unwanted errors

Reply:
The data is not dynamic here, I think we don't need that